### ISSUE TITLE
Drag/Drop events for KeyUp / KeyDown

### ISSUE THREAD

**Issue by robert-altom** [robert-altom/test#342]
Events only react to Mouse0

**Event System**

`OnInitializePotentialDrag, OnBeginDrag, OnDrag, OnDrop, OnEndDrag`

`OnInitializePotentialDrag`  - raises when mouse0 is down

`OnBeginDrag` - raises onmuse0down  (if mouse0 is down more than 1 frame) 

`OnDrag` - When dragging is occurring this will be called every time the cursor is moved.

`OnDrop` - raises on mouse0up on the element intersected by raycast. ( raises before OnEndDrag)
`OnEndDrag` - raises on mouse0up on the element that was dragged.

**Monobehaviour**

`OnMouseDrag` 

OnMouseDrag is called when the user has clicked on a Collider and is still holding down the mouse.

OnMouseDrag is called every frame while the mouse is down.

---
<sub>You can find the original issue from GitLab [here](https://gitlab.com/altom/altunity/altunitytester/-/issues/567).</sub>

**Comment by robert-altom** [comment]
Re-Tested with Inspector v.1.3.14 and game: Samplescenes for Windows 

- :heavy_check_mark: Checked that the drag action can be performed by using the left-click, right-click and wheel-click of the mouse (with action `Click/Press`)
- :heavy_check_mark: Checked that with `Click/Press` the draggable panels move only after clicking on them with mouse drag (left-click/ right-click/ wheel-click)

The issue is fixed. Closing the ticket.

**Comment by robert-altom** [comment]
Tested with Inspector v.1.3.14 and game: Samplescenes for Windows 

- :heavy_check_mark: Checked that the drag action can be performed by using the left-click, right-click and wheel-click of the mouse (with action `Click/Press`)
- **BUT** found another issue: :exclamation: when the `Click/Press` action is selected and mouse hovering on the screenshot area :arrow_forward: the draggable elements are moved along with the mouse hovering
  - **Steps**:
    - 1. After connecting the game, select _Live Update_ ON and action `Click/Press`
    - 2. From Manage Scene modal select "Scene 3 Drag and Drop"
    - 3. Hover the mouse on the screenshot area and check the draggable panels' motion.
  - **Actual result**:
    - In step 3 the draggable panels move along with the mouse hovering 
     ![Recording__153](https://gitlab.com/Altom/AltUnity/AltUnityTester/uploads/4206514396cbe7a4f28052807782a5b9/Recording__153.mp4)
   
  - **Expected result**:
    - In step 3 the draggable panels should move only after clicking on them with mouse drag

For the above mentioned issue will reopen the ticket.
